Firebase/ios/Bonfire/Mobile/Welcome/WelcomeView.swift

```diff
@@ -2,16 +2,19 @@ import UIKit
 import RxSwift
 import RxCocoa
 
+
 final class WelcomeView: UIView {
 
-    private let senderProfileImage = UIImageView()
-    private let senderLabel = UILabel()
+    private let bubblesBackground = BubbleBackgroundView()
+    private var senderInfoView = SenderInfoView()
     private let welcomeMessageLabel = UILabel()
     private let loginButton = UIButton()
 
     weak var actionListener: WelcomeActionListener?
 
-    private let disposeBag = DisposeBag()
+    private var disposeBag = DisposeBag()
+    private let actionDisposeBag = DisposeBag()
```

Comment:
It might be wroth renaming `disposeBag` to 'imageRequestDisposeBag` or something similar - to make it clear not to add any longstanding subscriptions to it